responses/responses.json

```diff
@@ -151,5 +151,29 @@
     "list_of_words": ["do", "you", "want", "to", "have", "sex", "make", "love"],
     "is_single_response": false,
     "required_word": ["sex"]
+  },
+
+  "your_job": {
+    "response": ["My job is to chatting with you all of course."],
+    "list_of_words": ["what","is","your","job","purpose"],
+    "is_single_response": false,
+    "required_word": []
+  },
+
+  "another_world": {
+    "response": [
+      "I don't know how to go to the another world but, you can ask the truck driver near that building corner."
```

Comment:
This should be "I don't know how to go to the another world. But you can ask the truck driver near that building corner."